content/concepts/introduction/protocols/identify.md

```diff
@@ -4,4 +4,40 @@ description: "A way for nodes in libp2p to query and inform other nodes about th
 weight: 23
 ---
 
-Coming soon!
+The libp2p Identify protocol is a way for nodes in the libp2p network to share
+and query information about each other. This information includes:
+
+- the user agent: a free-form string identifying the node's implementation, usually
+  in the format "agent-name/version".
+- the node's public IP address: the address of the node on the network, as seen by other nodes.
+- the list of multiaddresses that the node is listening on: the various network addresses
+  and ports that the node can be reached at.
+- the list of protocols that the node supports: the different communication methods that
+  the node can understand and use to communicate with other nodes.
+
+There are two variations of the protocol: `identify` and `identify/push`.
+
+The Identify protocol, identified by the protocol ID `/ipfs/id/1.0.0`, is used
+to query a remote node for its metadata. To do so, a node opens a stream to the
+remote node and the remote node responds with an `Identify` message containing
+its metadata and closes the stream.
+
+<!-- ADD Diagram -->
```

Comment:
Not sure if we need one, this should be really straightforward.

Reply:
I had sketched simple diagrams, one for identity and identify/push.